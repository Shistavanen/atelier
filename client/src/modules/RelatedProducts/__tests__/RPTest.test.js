// import React from 'react';
// import renderer from 'react-test-renderer';
// import RelatedProducts from '../RelatedProducts';

// it('RelatedProducts matches snapshot', () => {
//   const tree = renderer.create(<RelatedProducts />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
