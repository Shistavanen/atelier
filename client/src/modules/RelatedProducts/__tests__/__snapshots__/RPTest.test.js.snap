// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedProducts matches snapshot 1`] = `
<div>
  RELATED PRODUCTS
  <div
    className="container"
    onClick={[Function]}
  >
    <i
      className="fa fa-star-o"
    />
  </div>
  <div>
    <img
      src=""
    />
  </div>
  CATEGORY
  <div />
  <div />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
  <div>
    YOUR OUTFIT
  </div>
  <div
    className="container"
    onClick={[Function]}
  >
    <i
      className="fa fa-times"
    />
  </div>
  <div>
    <img
      src=""
    />
  </div>
  CATEGORY
  <div />
  <div />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
  <i
    className="fa fa-star-o"
  />
</div>
`;
